lib/repo/git_repository.rb

```diff
@@ -30,15 +30,15 @@ class GitRepository < Repository::AbstractRepository
     # created using GitRespository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
+      if IS_REPOSITORY_ADMIN.nil?
```

Comment:
I don't have a strong preference between globals or MarkUsConfiguator, both are better than Repoistory.conf which doesn't work in some situations and never works with Travis, I'll update the file to use MarkUsConfiguator.